qa/multi-cluster-search/src/test/java/org/elasticsearch/search/CCSDuelIT.java

```diff
@@ -955,25 +940,23 @@ public void testCompletionSuggester() throws Exception {
         suggestBuilder.addSuggestion("java", new CompletionSuggestionBuilder("suggest").size(20).text("jav"));
         suggestBuilder.addSuggestion("ruby", new CompletionSuggestionBuilder("suggest").size(30).text("rub"));
         sourceBuilder.suggest(suggestBuilder);
-        Consumer<SearchResponse> responseChecker = response -> {
-            assertEquals(Strings.toString(response, true, true), 3, response.getSuggest().size());
-            assertThat(response.getSuggest().getSuggestion("python").getEntries().size(), greaterThan(0));
-            assertThat(response.getSuggest().getSuggestion("java").getEntries().size(), greaterThan(0));
-            assertThat(response.getSuggest().getSuggestion("ruby").getEntries().size(), greaterThan(0));
+        CheckedConsumer<ObjectPath, IOException> responseChecker = response -> {
+            assertThat(response.evaluateMapKeys("suggest").size(), equalTo(3));
+            assertThat(response.evaluateArraySize("suggest.completion#python"), greaterThan(0));
+            assertThat(response.evaluateArraySize("suggest.completion#java"), greaterThan(0));
+            assertThat(response.evaluateArraySize("suggest.completion#ruby"), greaterThan(0));
         };
         {
             SearchRequest searchRequest = initLocalAndRemoteSearchRequest();
             searchRequest.source(sourceBuilder);
-            responseChecker.andThen(CCSDuelIT::assertMultiClusterSearchResponse);
             // suggest-only queries are not supported by _async_search, so only test against sync search API
-            duelSearchSync(searchRequest, responseChecker);
+            duelSearchSync(searchRequest, responseChecker.andThen(CCSDuelIT::assertMultiClusterSearchResponse));
         }
         {
             SearchRequest searchRequest = initRemoteOnlySearchRequest();
             searchRequest.source(sourceBuilder);
-            responseChecker.andThen(CCSDuelIT::assertSingleRemoteClusterSearchResponse);
```

Comment:
This method only returns a new Consumer so it is a mistake here? I corrected in the new version so the new Consumer is used for testing